lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -1522,6 +1531,16 @@ void DAP::EventThread() {
   }
 }
 
+void DAP::HandleThreadEvent(const lldb::SBEvent &event) {
```

Comment:
Is there a reason this isn't handled inline like the other events? Or do we want to break things down to make the EventThread more simple?